Packs/CVESearch/Integrations/CVESearch/CVESearch_test.py

```diff
@@ -0,0 +1,44 @@
+from CVESearch import *
+import demistomock as demisto
+
+
+def test_wrong_path(mocker):
+    _patch_mocks(mocker, "http://cve.bad_url", "cve", {"cve_id": "CVE-2014-1234"})
+    try:
+        main()
+        assert False, "Bad url- Exception should by raised"
+    except:  # noqa: E722
+        pass
+    results = _get_results()
+    expected_exception_message = "Verify that the server URL parameter is correct"
+    assert expected_exception_message in results, "Bad error response when bad url is given"
+
+
+def test_module(mocker):
+    _patch_mocks(mocker, "http://cve.circl.lu/api/", "cve-latest", {})
+    main()
+    results = _get_results()
+    assert len(results) == 30, "'cve-latest' command should return latest 30 cve's"
+
+
+def test_bad_cve_id(mocker):
+    bad_cve_id = "CVE-bad-cve"
+    _patch_mocks(mocker, "http://cve.circl.lu/api/", "cve", {"cve_id": bad_cve_id})
+    try:
+        main()
+        assert False, "Bad url- Exception should by raised"
+    except:  # noqa: E722
+        pass
+    results = _get_results()
+    assert results == f'Could not find cve results for ID {bad_cve_id}'
+
+
+def _patch_mocks(mocker, url, command, args):
+    mocker.patch.object(demisto, 'args', return_value=args)
```

Comment:
no need to patch all these for all functions, you can just call the relevant tested function